package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Google Labs repo for rapid prototyping",
   "type": "module",
   "scripts": {
-    "sync": "npm i && npm run clean:build && npm run build",
+    "clean": "git clean -fdx",
+    "sync": "npm run clean && npm install && npm run build",
+    "purge": "npm run clean && npx rimraf -g package-lock.json",
```

Comment:
How about instead of `purge` we have `"update": "rimraf package-lock.json && npm i"`? That's what refreshing the package-lock really does (within the semver constraints), and that seems like a useful thing to do independent of cleaning.

Nits:
- Don't need `npx` because all npm scripts get npm bins in `$PATH` automatically.
- Don't need `-g` because it's not a glob

Reply:
```suggestion
    "update": "rimraf package-lock.json && npm i",
```